### PR TITLE
Adding Summer 2019 Ebike Spd Controller Blog Post

### PR DESCRIPTION
**Pull request by tzmetz**

### CODE REVIEW

**Comment by tzmetz** [comment]
Just waiting on the link for the paper

File: content/EbikeSpdControllerBMDPresentationBlogPost.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+From September 9th to the 11th, the design and implementation of the electric bicycle speed controller project was presented in a poster exhibition hall during the 2019 Bicycle and Motorcycle Dynamics Conference held in Padova, Italy. 
+
+Below you can find the poster presented at the conference.
+
+<iframe src="https://widgets.figshare.com/articles/9883154/embed?show_title=1" width="568" height="351" allowfullscreen="true" frameborder="0"></iframe> 
+
+Find the poster `here <https://doi.org/10.6084/m9.figshare.9883154.v1>`_. 
+
+Read the conference paper `here <link>`_.
```
Comment (moorepants) [inline content/EbikeSpdControllerBMDPresentationBlogPost.txt]:
```suggestion
Read the conference paper `here <https://doi.org/10.6084/m9.figshare.9937091.v1>`_. 
```

**Comment by moorepants** [comment]
Suggested the link above.

**Comment by tzmetz** [comment]
Looks ready to publish from my end

**Comment by moorepants** [comment]
It's in! Thanks.

**Pull request merged**